recipes/fft/all/CMakeLists.txt

```
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(fft LANGUAGES C)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+option(THREADS "use threads")
+
+set(SOURCES "fft_build.c")
+set(HEADERS "alloc.h;fft.h;fft2.h;fft3.h;dct.h;${CMAKE_CURRENT_BINARY_DIR}/fft_export.h")
+
+add_library(${PROJECT_NAME} ${SOURCES} ${HEADERS})
+
+set_target_properties(${PROJECT_NAME} PROPERTIES PUBLIC_HEADER "${HEADERS}")
```

```suggestion
set_target_properties(${PROJECT_NAME} PROPERTIES
  PUBLIC_HEADER "${HEADERS}"
  C_VISIBILITY_PRESET hidden)
```

Let's hide symbols since you took time to add generate_export_header